src/WebJobs.Script/Rpc/LanguageWorkerConstants.cs

```diff
@@ -24,5 +23,9 @@ public static class LanguageWorkerConstants
         public const string WorkerDescriptionDefaultWorkerPath = "defaultWorkerPath";
         public const string WorkerDescription = "Description";
         public const string WorkerDescriptionArguments = "Arguments";
+
+        // Java
+        public const string AppServiceEnvJavaVersion = "zulu8.23.0.3-jdk8.0.144-win_x64";
```

Comment:
This hard coded version seems brittle. What if someone specifies their own java_home value pointing to their own version that is different?

Reply:
Opened https://github.com/Azure/azure-functions-host/issues/3048 to update worker.config.json to provide java version